README.md

```diff
@@ -495,11 +488,19 @@ absolute path as on the host.
 By default, Docker container restricts the access to `perf_event_open`
 syscall. There are 3 alternatives to allow profiling in a container:
 1. You can modify the [seccomp profile](https://docs.docker.com/engine/security/seccomp/)
-or disable it altogether with `--security-opt seccomp=unconfined` option. In
-addition, `--cap-add SYS_ADMIN` may be required.
+   or disable it altogether with `--security-opt seccomp=unconfined` option. In
+   addition, `--cap-add SYS_ADMIN` may be required.
 2. You can use "fdtransfer": see the help for `--fdtransfer`.
 3. Last, you may fall back to `-e ctimer` profiling mode, see [Troubleshooting](#troubleshooting).
 
+## Building
```

Comment:
We can mention min. supported JDK version and C++ standard here, with minimum gcc and clang versions.